print-bgp.c

```diff
@@ -2766,6 +2795,15 @@ bgp_capabilities_print(netdissect_options *ndo,
                 cap_offset += 4;
             }
             break;
+        case BGP_CAPCODE_ROLE:
+            if (cap_len < 1) {
+                ND_PRINT(" (too short, < 1)");
+                return;
+            }
+            ND_PRINT("\n\t\t Role name %s",
+                     tok2str(bgp_role_values, "Unassigned",
```

Comment:
Do you want to use something like "Unassigned (%d)" so that we know what the numeric value was?

Reply:
Thanks, fixed.